### PR TITLE
Update Findings page docs - specify procedures for Group by rule and No grouping views

### PR DESCRIPTION
**Pull request by adamkvitek**
# Thanks for improving Semgrep Docs 😀

### Please ensure

- [x] A subject matter expert (SME) reviews the content
- [x] A technical writer reviews the content or PR
- [ ] This change has no security implications or else you have pinged the security team
- [ ] Redirects are added if the PR changes page URLs
- [ ] If you have changed any header tag links (doc/#this-kind-of-anchor), update all instances of that link

### CODE REVIEW

File: docs/ignoring-files-folders-code.md [hunk 1]
```diff
@@ -106,9 +106,9 @@ To define files and folders in Semgrep Cloud Platform:
 
 Including files and folders through this method is **additive**. When Semgrep Cloud Platform makes a scan, it looks for a `.semgrepignore` within the repository. If no `.semgrepignore` file is found, Semgrep temporarily creates one and adds items from Semgrep Cloud Platform's Path Ignores. Adding items to the **Path Ignores** box does not override default Semgrep ignore patterns.
 
-You can also add files to `.semgrepignore` while triaging your findings by issuing the following steps:
+You can also add files to `.semgrepignore` while triaging individual findings in **No grouping** view on the Findings page:
```
Comment (pabloest) [inline docs/ignoring-files-folders-code.md:109]:
```suggestion
You can also add files to `.semgrepignore` while triaging individual findings in the **No grouping** view on the Findings page:
```

File: docs/semgrep-code/findings.md [hunk 2]
```diff
@@ -174,9 +174,20 @@ To manage, change, open or ignore findings, follow the the triage processes desc
 
 #### Ignoring findings
 
-##### Ignoring multiple findings
+##### Ignoring findings in Group by Rule view
 
-To **ignore multiple findings**, follow these steps:
+To **ignore findings** in **Group by Rule** view, follow these steps:
```
Comment (pabloest) [inline docs/semgrep-code/findings.md:179]:
```suggestion
To **ignore findings** in the **Group by Rule** view, follow these steps:
```

File: docs/semgrep-code/findings.md [hunk 3]
```diff
@@ -174,9 +174,20 @@ To manage, change, open or ignore findings, follow the the triage processes desc
...
+
+1. On the [Findings](https://semgrep.dev/orgs/-/findings?tab=open) page, click the **Status** filter, and then select **Open** status to see all open findings.
+1. Perform one of these steps:
+    - To select more findings from the same rule, click **Triage** button on the card of the finding.
```
Comment (pabloest) [inline docs/semgrep-code/findings.md:183]:
```suggestion
    - To select more findings from the same rule, click the **Triage** button on the card of the finding.
```

File: docs/semgrep-code/findings.md [hunk 4]
```diff
@@ -174,9 +174,20 @@ To manage, change, open or ignore findings, follow the the triage processes desc
...
+    - To select individual findingss reported by a rule, fill in the checkboxes of the finding, and then click **Triage** button on the card of the finding.
```
Comment (pabloest) [inline docs/semgrep-code/findings.md:184]:
```suggestion
    - To select individual findingss reported by a rule, fill in the checkboxes of the finding, and then click the **Triage** button on the card of the finding.
```

File: docs/semgrep-code/findings.md [hunk 5]
```diff
@@ -174,9 +174,20 @@ To manage, change, open or ignore findings, follow the the triage processes desc
...
+1. Optional: Write a why is the finding ignored.
```
Comment (pabloest) [inline docs/semgrep-code/findings.md:185]:
```suggestion
1. Optional: Write a reason to describe why the finding was ignored.
```

File: docs/semgrep-code/findings.md [hunk 6]
```diff
@@ -174,9 +174,20 @@ To manage, change, open or ignore findings, follow the the triage processes desc
...
+1. Click **Ignore**.
+
+##### Ignoring findings in No grouping view
```
Comment (pabloest) [inline docs/semgrep-code/findings.md:188]:
```suggestion
##### Ignoring findings in the No grouping view
```

File: docs/semgrep-code/findings.md [hunk 7]
```diff
@@ -174,9 +174,20 @@ To manage, change, open or ignore findings, follow the the triage processes desc
...
+
+To **ignore multiple findings** in **No grouping** view, follow these steps:
```
Comment (pabloest) [inline docs/semgrep-code/findings.md:190]:
```suggestion
To **ignore multiple findings** in the **No grouping** view, follow these steps:
```

File: docs/semgrep-code/findings.md [hunk 8]
```diff
@@ -187,21 +198,28 @@ To **ignore multiple findings**, follow these steps:
 1. Select **Ignored** from the dropdown menu.
 1. Click **Save**.
 
-##### Ignoring individual findings
-
-To **ignore individual findings**, follow these steps:
+##### Ignoring individual findings in No grouping view
 
-<IgnoreIndividualFindings />
+To **ignore individual finding** in **No grouping** view, follow these steps:
```
Comment (pabloest) [inline docs/semgrep-code/findings.md:203]:
```suggestion
To **ignore individual finding** in the **No grouping** view, follow these steps:
```

File: docs/semgrep-code/findings.md [hunk 9]
```diff
@@ -187,21 +198,28 @@ To **ignore multiple findings**, follow these steps:
...
-If you ignore all findings in **Just this file**, **This directory**, or **Parent directory** in the fifth step of the procedure above, these files or directories are added to `.semgrepignore` file. For more information, see [Ignoring files, folders, or parts of code](/ignoring-files-folders-code/).
-:::
+<IgnoreIndividualFindingNoGrouping />
 
 #### Reopening findings
 
-##### Reopening multiple findings
+##### Reopening findings in Group by Rule view
+
+To **reopen findings** in **Group by Rule** view, follow these steps:
```
Comment (pabloest) [inline docs/semgrep-code/findings.md:211]:
```suggestion
To **reopen findings** in the **Group by Rule** view, follow these steps:
```

File: docs/semgrep-code/findings.md [hunk 10]
```diff
@@ -187,21 +198,28 @@ To **ignore multiple findings**, follow these steps:
...
+
+1. On the [Findings](https://semgrep.dev/orgs/-/findings?tab=open) page, click the **Status** filter, and then select **Ignored** or **Fixed** status to see all ignored or fixed findings.
```
Comment (pabloest) [inline docs/semgrep-code/findings.md:213]:
```suggestion
1. On the [Findings](https://semgrep.dev/orgs/-/findings?tab=open) page, click the **Status** filter, and then select the **Ignored** or **Fixed** status to see all ignored or fixed findings.
```

File: docs/semgrep-code/findings.md [hunk 11]
```diff
@@ -187,21 +198,28 @@ To **ignore multiple findings**, follow these steps:
...
+1. Perform one of these steps:
+    - To select more findings from the same rule, click **Triage** button on the card of the finding.
```
Comment (pabloest) [inline docs/semgrep-code/findings.md:215]:
```suggestion
    - To select more findings from the same rule, click the **Triage** button on the card of the finding.
```

File: docs/semgrep-code/findings.md [hunk 12]
```diff
@@ -187,21 +198,28 @@ To **ignore multiple findings**, follow these steps:
...
+    - To select individual findingss reported by a rule, fill in the checkboxes of the finding, and then click **Triage** button on the card of the finding.
```
Comment (pabloest) [inline docs/semgrep-code/findings.md:216]:
```suggestion
    - To select individual findingss reported by a rule, fill in the checkboxes of the finding, and then click the **Triage** button on the card of the finding.
```

File: docs/semgrep-code/findings.md [hunk 13]
```diff
@@ -187,21 +198,28 @@ To **ignore multiple findings**, follow these steps:
...
+1. Optional: Write why is the finding reopened.
```
Comment (pabloest) [inline docs/semgrep-code/findings.md:217]:
```suggestion
1. Optional: Write a reason to describe why the finding was ignored.
```

File: docs/semgrep-code/findings.md [hunk 14]
```diff
@@ -187,21 +198,28 @@ To **ignore multiple findings**, follow these steps:
...
+1. Click **Reopen**.
+
+##### Reopening multiple findings in No grouping view
```
Comment (pabloest) [inline docs/semgrep-code/findings.md:220]:
```suggestion
##### Reopening multiple findings in the No grouping view
```

File: docs/semgrep-code/findings.md [hunk 15]
```diff
@@ -187,21 +198,28 @@ To **ignore multiple findings**, follow these steps:
...
 
-To **open findings**, follow these steps:
+To **open findings** in **No grouping** view, follow these steps:
```
Comment (pabloest) [inline docs/semgrep-code/findings.md:222]:
```suggestion
To **open findings** in the **No grouping** view, follow these steps:
```

File: docs/semgrep-code/findings.md [hunk 16]
```diff
@@ -187,21 +198,28 @@ To **ignore multiple findings**, follow these steps:
...
 
 1. On the [Findings](https://semgrep.dev/orgs/-/findings?tab=open) page, click the **Status** filter, and then select **Ignored** or **Fixed** status to see all ignored or fixed findings.
```
Comment (pabloest) [inline docs/semgrep-code/findings.md:224]:
```suggestion
1. On the [Findings](https://semgrep.dev/orgs/-/findings?tab=open) page, click the **Status** filter, and then select the **Ignored** or **Fixed** status to see all ignored or fixed findings.
```

File: docs/semgrep-code/findings.md [hunk 17]
```diff
@@ -211,7 +229,7 @@ To **open findings**, follow these steps:
 1. In the **Triage state** dropdown menu, select **Reopened**.
 1. Click **Save**.
 
-##### Reopening individual findings
+##### Reopening individual findings in No grouping view
```
Comment (pabloest) [inline docs/semgrep-code/findings.md:232]:
```suggestion
##### Reopening individual findings in the No grouping view
```

File: src/components/procedure/_ignore-individual-finding-no-grouping.mdx [hunk 18]
```diff
@@ -0,0 +1,7 @@
+1. On the Semgrep Code [Findings](https://semgrep.dev/orgs/-/findings?tab=open) page, click the **Status** filter, and then select **Open** status to see all open findings.
+1. Next to a finding you want to ignore, click the **Open**.
```
Comment (pabloest) [inline src/components/procedure/_ignore-individual-finding-no-grouping.mdx:2]:
```suggestion
1. Next to a finding you want to ignore, click **Open**.
```
?

**Review by pabloest — APPROVED** [review]

File: src/components/procedure/_ignore-individual-finding-no-grouping.mdx (hunk 18 above)
Comment (adamkvitek) [reply src/components/procedure/_ignore-individual-finding-no-grouping.mdx]: @pabloest true, thanks for the suggestion (it is also part of our style guide).

**Review by raghavjain3 — APPROVED** [review]

**Pull request merged**